client.cfg

```
@@ -154,6 +160,16 @@ gcode:
     {% endif %}
   {% else %}
     RESPOND TYPE=error MSG='{"Resume aborted !!! \"%s\" detects no filament, please load filament and press RESUME" % (client.runout_sensor.split(" "))[1]}'
+    {% set _d = prompt_txt.append("\"%s\" detects no filament, please load filament and press RESUME" % (client.runout_sensor.split(" "))[1]) %}
+  {% endif %}
+  ##### Generate User Information box in case of abort #####
+  {% if not (runout_resume and do_resume) %} 
+    RESPOND TYPE=command MSG="action:prompt_begin RESUME aborted !!!"
+    {% for element in prompt_txt %}
+      RESPOND TYPE=command MSG='{"action:prompt_text %s" % element}' 
+    {% endfor %}
+    RESPOND TYPE=command MSG="action:prompt_footer_button Ok|RESPOND TYPE=command MSG="action:prompt_end"|info"
```

I'm surprised Klipper can parse this as the quotes here seem a bit incorrect to me.

I think we can even just remove the inner quotes and it will work as expected.

```suggestion
    RESPOND TYPE=command MSG="action:prompt_footer_button Ok|RESPOND TYPE=command MSG=action:prompt_end|info"
```

---

Yes they should not be in there